docs/notebooks/image_to_image_arcface.md

```diff
@@ -0,0 +1,224 @@
+# Image-to-Image Search with ArcFaceLoss
+
+<a href="https://colab.research.google.com/drive/1ZS9FmnR9FzO_JYGCPazFM7TcMNQl51xM?usp=sharing"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg"></a>
+
+Searching visually similar images with image queries is a very popular use case. However, using pre-trained models does not deliver the best results – the models are trained on general data that lack the particularities of your specific task. Here's where Finetuner comes in! It enables you to accomplish this easily.
+
+This guide will demonstrate how to fine-tune a ResNet model using the `ArcFaceLoss` function.
+
+*Note, please consider switching to GPU/TPU Runtime for faster inference.*
+
+## Install
+<!-- #endregion -->
+
+```python id="VdKH0S0FrwS3"
+!pip install 'finetuner[full]'
+```
+
+<!-- #region id="7EliQdGCsdL0" -->
+## Task
+
+More specifically, we will fine-tune ResNet50 on [Stanford Cars Dataset](http://ai.stanford.edu/~jkrause/cars/car_dataset.html).
+This dataset consists of 196 classes across 16184 docuemnts in total.
+
+Each class of the dataset consists of roughly 82 documents, in order to group the documents of each class together and move separate documents of different classes apart, we use the `ArcFaceLoss` function. For more information on how this loss function works, see [Advanced Losses and Optimizers](https://finetuner.jina.ai/advanced-topics/advanced-losses-and-optimizers/)
+
+After fine-tuning, documents from each class should have similar embeddings, distinct from documents of other classes.
+<!-- #endregion -->
+
+<!-- #region id="M1sii3xdtD2y" -->
+## Data
+
+Our journey starts locally. We have to prepare the data and push it to the Jina AI Cloud and Finetuner will be able to get the dataset by its name. For this example,
+we already prepared the data, and we'll provide the names of training data (`stanford-cars-train`) directly to Finetuner.
+
+```{important} 
+We don't require you to push data to the Jina AI Cloud by yourself. Instead of a name, you can provide a `DocumentArray` and Finetuner will do the job for you.
+When working with documents where images are stored locally, please call `doc.load_uri_to_blob()` to reduce network transmission and speed up training.
+```
+<!-- #endregion -->
+
+```python id="L0NfPGbTkNsc"
+import finetuner
+from docarray import DocumentArray, Document
+
+finetuner.login(force=True)
+```
+
+```python id="ONpXDwFBsqQS"
+train_data = DocumentArray.pull('finetuner/stanford-cars-train', show_progress=True)
+query_data = DocumentArray.pull('finetuner/stanford-cars-query', show_progress=True)
+index_data = DocumentArray.pull('finetuner/stanford-cars-index', show_progress=True)
+
+train_data.summary()
+```
+
+<!-- #region id="mUoY1jq0klwk" -->
+## Backbone model
+Now let's see which backbone models we can use. You can see available models by calling `finetuner.describe_models()`.
+
+
+For this example, we're gonna go with `resnet50`.
```

Comment:
The other image-to-image example also misses this but we could add that this model is trained on the ImageNet classification task and it will be adapted by finetuner into an embedding model which is trained in the next step